From: Sebastian Ramacher <sramacher@debian.org>
Date: Mon, 19 Nov 2018 22:35:07 +0100
Subject: Do not build cmrtlib

---
 CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 77f7416..9be262e 100755
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -42,8 +42,6 @@ endif()
 
 include(GNUInstallDirs)
 
-add_subdirectory(cmrtlib)
-
 add_subdirectory(media_driver)
 
 if("${LIBVA_DRIVERS_PATH}" STREQUAL "")
@@ -52,7 +50,6 @@ endif()
 message(STATUS "LIBVA_DRIVERS_PATH = ${LIBVA_DRIVERS_PATH}")
 
 install (FILES ${CMAKE_CURRENT_BINARY_DIR}/media_driver/iHD_drv_video.so DESTINATION ${LIBVA_DRIVERS_PATH} COMPONENT media)
-install (FILES ${CMAKE_CURRENT_BINARY_DIR}/cmrtlib/linux/igfxcmrt64.so DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT media)
 
 option (INSTALL_DRIVER_SYSCONF "Install driver system configuration file" ON)
 if (INSTALL_DRIVER_SYSCONF)
